### DISCUSSION TITLE
最近B站首页在灰度测试个性换推荐，能换回之前的全区推荐吗

### DISCUSSION THREAD

**Discussion by link-ping** [the1812/Bilibili-Evolved#1842] [功能建议 / Ideas]
![j1](https://user-images.githubusercontent.com/65518238/117924205-f165a900-b327-11eb-939d-65bc45e556eb.png)
这是登录了账号的首页推荐，只有个换一换按钮了
![j2](https://user-images.githubusercontent.com/65518238/117924234-ffb3c500-b327-11eb-827d-56bc280d27e1.png)
这是没登录账号的情况，还是和之前的一样

**Comment by Geeyun-JY3** [comment]
这玩意测试很久了，你应该是最近刚被灰度到

**Comment by the1812** [comment]
https://greasyfork.org/zh-CN/scripts/425943-%E7%A6%81%E7%94%A8b%E7%AB%99%E9%A6%96%E9%A1%B5%E9%A6%96%E6%A0%8F%E4%B8%AA%E6%80%A7%E6%8E%A8%E8%8D%90

**Reply by x486** [reply]
可惜这个脚本不能翻页

**Reply by ShuiHuo** [reply]
开了简化首页后这个脚本就不起作用了
